### ISSUE TITLE
custom dataset

### ISSUE THREAD

**Issue by MaleshwarSastri** [phecy/SSL-FEW-SHOT#7]
Hey!
I am trying to train my own custom dataset. How should i train and evaluate it?
please also provide few details on evaluate protonet.

**Comment by Lyrics1988** [comment]
Hi,

Thanks for your interest in our work.

For a custom dataset, you will need to pre-train the network in stage 1 with self-supervised learning following the AMDIM method(Github link provided) and fine-tune it with meta-learning(stage2)

**Comment by mohaEs** [comment]
Sorry, I am a little confused about how to use labels. I've found that you are using torch.arange. I don't know how it will be linked to the dataset. May I ask for a hint?
